documentation/learningPath/egress.md

```diff
@@ -0,0 +1,77 @@
+
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2FlearningPath%2Fegress)
+
+# Egress
+
+`dotnet monitor` includes functionality to egress (send) artifacts to permanent storage locations, such as `Azure Blob Storage`. For .NET Monitor 8, the tool has converted to using an extensible egress model that allows developers to author their own egress providers that aren't built into the `dotnet monitor` product. This section covers how the egress extensibility model works, and provides information about how to develop an egress extension (using the `AzureBlobStorage` egress provider as an example). 
+
+## How Egress Works
+
+```mermaid
+%%{ init: { 'flowchart': { 'curve': 'basis' } } }%%
+graph LR
+    classDef altColor fill:#CAF,stroke:purple;
+    subgraph ide1 [.NET Monitor]
+    A[Configuration] --> N{.NET Monitor}
+    N --> 1
+    N --> 2   
+    N --> 3
+    end
+    subgraph ide2 [Extensions]
+    3 ---> R{AzureBlobStorage}
+    R --> 4
+    R --> 5
+    end
+    
+    class ide2 altColor
+```
+
+1. [User initiates collection of artifact with a designated egress provider](https://github.com/dotnet/dotnet-monitor/blob/289105261537f3977f7d1886f936d19bb3639d46/src/Microsoft.Diagnostics.Monitoring.WebApi/Operation/EgressOperation.cs#L49)
+1. [Locate extension's executable and manifest](https://github.com/dotnet/dotnet-monitor/blob/289105261537f3977f7d1886f936d19bb3639d46/src/Tools/dotnet-monitor/Extensibility/ExtensionDiscoverer.cs#L28)
+1. [Start extension and pass configuration/artifact via StdIn to the other process](https://github.com/dotnet/dotnet-monitor/blob/289105261537f3977f7d1886f936d19bb3639d46/src/Tools/dotnet-monitor/Egress/Extension/EgressExtension.cs#L102)
+1. [Connect to egress provider using configuration and send artifact](https://github.com/dotnet/dotnet-monitor/blob/289105261537f3977f7d1886f936d19bb3639d46/src/Extensions/AzureBlobStorage/AzureBlobEgressProvider.cs#L35)
+1. [Provide success/failure information via StdOut to dotnet-monitor](https://github.com/dotnet/dotnet-monitor/blob/289105261537f3977f7d1886f936d19bb3639d46/src/Microsoft.Diagnostics.Monitoring.Extension.Common/EgressHelper.cs#L77)
+
+
+## Distribution and Acquisition Model
+
+### Dotnet-Monitor Versions
+
+There are two versions of the `dotnet-monitor` tool being offered: `monitor` and `monitor-base`. The default version of `dotnet-monitor` includes every supported egress provider (to avoid breaking changes, this uses the existing `monitor` naming); the `monitor-base` version is a newly created image for .NET Monitor 8+ and only includes the `FileSystem` egress provider, allowing users to manually include their preferred egress providers.
```

Comment:
I know you are referring to 'tool' in the general sense, but this might be interpreted as dotnet tool. To my understanding we have a base image, but no base dotnet-monitor tool.